apps/transport/lib/irve/http_pagination.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Transport.IRVE.HTTPPagination do
+  @moduledoc """
+  Although namespaced under IRVE for simplicity, this is more data-gouv specific.
```

Comment:
On a `Datagouvfr.Client.API.stream` qui fait déjà ça, peut-on le réutiliser ?

Reply:
Dans les faits l'implémentation est différente même si le sujet est le même:
- utilisation de `Req` plutôt que `HTTPoison`, ce qui me permet d'utiliser le mécanisme de cache local déjà en place (utile pour itérer plus rapidement en dév local, de beaucoup), et aussi on a vu que la maintenance de `HTTPoison` est plus poussive actuellement
- génération de la totalité des requêtes de pagination en un seul coup, ce qui permet de paralléliser les téléchargements derrière (et m'a fait gagner du temps)
- pas d'utilisation de clé d'API data gouv

Il y a une redondance thématique on va dire, mais je préfère conserver en l'état, l'implémentation est plus performante (parallélisée), mise en cache triviale utile en dév, et client mieux maintenu.

Je trouve préférable cf maintenance de continuer à développer du code qui s'appuie sur `Req` qui semble plus pérenne que `HTTPoison` et moins chargé en historique.